macros/src/main/scala/packaged/insidePackageObject/package.scala

```diff
@@ -2,7 +2,7 @@ package packaged
 
 import scala.gestalt._
 
-package object macros2 {
+package object insidePackageObject {
   inline def plus(a: Int, b: Int): Int = meta {
     q"$a + $b"
   }
```

Comment:
I just try to compile the following file:

```Scala
import scala.gestalt._

package object A {
  inline def plus(a: Int, b: Int): Int = meta { q"$a + $b" }
}

package object B {
  inline def plus(a: Int, b: Int): Int =  a + b
}
```

Somehow in the translation of macro definitions, `A` looses the `package` flag. Seems to be some bug with the compiler. I'll propose a fix soon.